graphql_query_derive/src/lib.rs

```diff
@@ -37,7 +37,7 @@ fn build_graphql_client_derive_options(input: &syn::DeriveInput) -> GraphQLClien
         .unwrap_or(deprecation::DeprecationStrategy::default());
 
     GraphQLClientDeriveOptions {
-        selected_operation: input.ident.to_string(),
+        struct_name: input.ident.to_string(),
```

Comment:
I renamed selected_operation to struct_name because we are probably going to generate code for _all_ the operations in a query document in codegen other than the derive.